js/state/frameStateUtil.js

```diff
@@ -84,7 +84,16 @@ export function findIndexForFrameKey (frames, key) {
  * Obtains the frameProps index in the frames
  */
 export function getFramePropsIndex (frames, frameProps) {
-  return frames.findIndex(found => found.get('key') === frameProps.get('key'))
+  return frames.findIndex(found => {
+    try {
+      return found.get('key') === frameProps.get('key')
+    } catch (e) {
+      // todo
+      // catch '.get is not a function'
+      // this seems to happen when testing
+      // possibly dispatchToIPC related
```

Comment:
I think it is frameProps - added the following to the `catch` block

```
console.log(found.get && found.toJS() || found)
console.log(frameProps.get && frameProps.toJS() || frameProps)
```

and it shows this for frameProps:

```
Object
@@__IMMUTABLE_ITERABLE__@@: true
@@__IMMUTABLE_KEYED__@@: true
@@__IMMUTABLE_MAP__@@: true
__altered: false
_root: Object
  entries: Array[3]
    0: Array[2]
      0: "findbarShown"
      1: false
      length: 2
    1: Array[2]
    2: Array[2]
    length: 3
ownerID: Object
size: 3
```

Never happens outside of test mode. I'll look into it a bit more